### PR TITLE
[Merged by Bors] - fix(tests-http): json bats

### PR DESCRIPTION
**Pull request by pinkforest**
* json bats fix extra { in body

### CODE REVIEW

**Review by morenol — APPROVED** [review]

File: rust-connectors/sources/http/tests/get-test-json.bats [hunk 1]
```diff
@@ -30,6 +30,6 @@ teardown() {
     sleep 10
 
     run fluvio consume -o 1 --end-offset 1 -d $TOPIC
-    assert_output --partial '{"body": {"Hello, Fluvio! - ' 
+    assert_output --partial '{"body":"Hello, Fluvio! - '
```
Comment (morenol) [inline rust-connectors/sources/http/tests/get-test-json.bats:33]: I think that we should remove the initial `{`

**Review by morenol — CHANGES_REQUESTED** [review]

File: rust-connectors/sources/http/tests/get-test-json.bats (hunk 1 above)
Comment (wendyOn) [reply rust-connectors/sources/http/tests/get-test-json.bats:33]: it found an actual bug. the status should not be there with output_parts body

Comment (wendyOn) [reply rust-connectors/sources/http/tests/get-test-json.bats:33]: yeah Mondays... :innocent:

**Comment by wendyOn** [comment]
bors retry

**Comment by wendyOn** [comment]
:sweat_smile: 

bors r+

**Comment by wendyOn** [comment]
k3d / CI red herring retrying

bors retry

**Comment by wendyOn** [comment]
bors retry

**Pull request closed without merging**